frontend/src/components/loginForm.tsx

```diff
@@ -33,6 +35,14 @@ export const LoginForm = () => {
     setPassword(event.target.value);
   };
 
+  const handleGoogleOAuth = () => {
+    window.open('http://localhost:8000/auth/google', '_self')
+  }
+
+  const handleFaceBookOAuth = () => {
+    window.open('http://localhost:8000/auth/facebook', '_self')
+  }
```

Comment:
This works -- but if we change our div to an `a` tag and then use the href and target field.

Reply:
Do we even need to navigate to a different page though? Doesn't the google/fb OAuth flow take you to like a "sign in via google / fb" page that you can then supply a success and fail redirect path?

Reply:
Yeah so I tried sending an GET request to the our backend google and facebook endpoint but it doesn't work and it gave me a CORS error. I think this is because OAuth doesn't support CORS ?  (https://stackoverflow.com/questions/55079181/what-is-the-best-practice-to-use-oauth2-react-node-js-and-passport-js-to-authe/63736799#63736799)

The only solution I've found so far is to do a redirect to the our OAuth endpoint directly.

Reply:
https://create-react-app.dev/docs/proxying-api-requests-in-development/

I was doing some more research and i think this is why we're not able to proxy our requests properly! Do you want to see if this solves our problem?

Reply:
I noticed that I didn't send you the error message. Here's an axios GET request to localhost:8000/auth/google (backend) gave me this error. This is for reference.

![image](https://user-images.githubusercontent.com/106122976/223188923-d2b52160-a2be-4c67-b5c2-025cfca6d07d.png)

Reply:
Also I tried proxying our requests to our backend by putting `"proxy": "http://localhost:8000"` in package.json as stated from the react doc you linked. Then I tried doing an axios GET request to`'http://localhost:8000/auth/google'`. I'm not sure how proxy works tbh.

![image](https://user-images.githubusercontent.com/106122976/223190959-a71df166-1613-46ba-be1f-a102a683cf68.png)

Reply:
I think if we have the proxy working, we don't need to specify `http://localhost:8000` anymore, just `/auth/google`.

The proxy works by sending any requests that originate in `localhost:3000` to `localhost:8000` (proxy server). If we're still getting CORS errors, then we might want to enable CORS in our headers if we need to.

Reply:
So I got the same CORS error message when I use `axios.get('/auth/google')` with `localhost:8000` as the proxy:

![image](https://user-images.githubusercontent.com/106122976/223332907-32d3e63d-8042-4a5f-8cfb-4e165c8a2980.png)

It seems like its trying to request to `localhost:3000/auth/google` instead of `localhost:8000/auth/google` with the proxy turn on. 

Network:
![image](https://user-images.githubusercontent.com/106122976/223333811-a19b45d8-4e40-4c3e-b74c-e3398a4eb8a9.png)